Makefile.devel

```diff
@@ -309,17 +293,17 @@ distcompile: FORCE
 
 ############################################################
 ##
-## tag:    tag the CVS sources of working directory with CVS_RELEASENAME,
-##	   set version stamp in proof-site.el and ProofGeneral.spec 
+## tag:    tag the sources of working directory with GIT_RELEASENAME,
+##	   set version stamp in proof-site.el and ProofGeneral.spec
 ##         to VERSION, and edit $(DOWNLOADHTMLS)
 ##	   if VERSION matches PRERELEASE_TAG.
 ##
-tag:		
+tag:
 	@echo "*************************************************"
 	@echo " Tagging sources... (you must rerun if CVS source dirty)"
 	@echo "*************************************************"
 # Update the sources, this is almost always what we want to do.
-	if [ -z "$(NOCVS)" ] && [ -n "`cvs -n -q update -Pd | grep '^[MC] '`" ]; then cvs update -Pd; exit 1; fi
+	if [ -z "$(NOCVS)" ]; then git status | grep '^Your branch is up-to-date with.*master' && git status | grep 'nothing to commit, working tree clean' || exit 1 fi
```

Comment:
This line should be replaced with:
`if [ -z "$(NOCVS)" ]; then git fetch origin && git log --exit-code master...origin/master || exit 1; fi`